cli/lsp/resolver.rs

```diff
@@ -484,8 +413,8 @@ fn create_graph_resolver(
     bare_node_builtins_enabled: config_file
       .map(|cf| cf.has_unstable("bare-node-builtins"))
       .unwrap_or(false),
-    // not used in the LSP as the LspGraphResolver handles this
-    sloppy_imports_resolver: None,
+    sloppy_imports_resolver: unstable_sloppy_imports
+      .then(|| SloppyImportsResolver::new(Arc::new(deno_fs::RealFs))),
```

Comment:
👍thanks! I went straight to this line when I saw this PR. The LspGraphResolver was created because I initially thought we should take into account the open documents, but this is much better.